CONTRIBUTING.md

```diff
@@ -134,6 +135,47 @@ cd test/manual/k8s_crds
 ```
 This test currently does not run as part of the test suite.
 
+## Documentation
+Secretless has a few sources for documentation: a website, a documentation subdomain, and godocs.
```

Comment:
I would be confused here what the difference is between "the website" and "the documentation subdomain".  I know this is explained below but it would be nice if the outlines of the differences could be conveyed here in a phrase.

Eg, "the website and the documentation subdomain are split for no good reason.  They are an aritfact of Conway's law"

:)